projects/shared/src/lib/directive/no-content.directive.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2023 RERO
+ * Copyright (C) 2023 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { AfterContentChecked, Directive, ElementRef, Input, Renderer2, TemplateRef, ViewContainerRef } from '@angular/core';
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: '[noContent]',
+})
+export class NoContentDirective implements AfterContentChecked {
+
```

Comment:
Can you put a comment to explain the purpose of this directive and adds some doc strings in the methods?